pkg/env/continuous_profiling.go

```diff
@@ -0,0 +1,18 @@
+package env
+
+var (
+	// ContinuousProfiling indicates if continuous profiling is enabled
+	ContinuousProfiling = RegisterBooleanSetting("ROX_CONTINUOUS_PROFILING", false)
+
+	// ContinuousProfilingServerAddress defines the server address for the continuous profiling
+	ContinuousProfilingServerAddress = RegisterSetting("ROX_CONTINUOUS_PROFILING_SERVER_ADDRESS")
+
+	// ContinuousProfilingBasicAuthUser defines the http basic auth user
+	ContinuousProfilingBasicAuthUser = RegisterSetting("ROX_CONTINUOUS_PROFILING_BASIC_AUTH_USER")
```

Comment:
Is there some default value used in pyroscope server? e.g., `admin`? maybe we could use it as a default value.

The same comment would apply to the password env var.

Reply:
The defaults for the user and password in pyroscope server are empty (so no authentication by default).